src/views/Header.js

```diff
@@ -99,11 +80,57 @@ class Header extends React.Component<DefaultProps, HeaderProps, *> {
     );
   }
 
+  _getHeaderTitle(navigation: Navigation): ?string {
+    const header = this.props.router.getScreenConfig(navigation, 'header');
+    if (header && header.title) {
```

Comment:
This needs to be `(header && typeof header.title === 'string')` to address empty string use case, and 0